test/image/mocks/14.json

```diff
@@ -2,432 +2,432 @@
             "type": "scatter"
         }
-    ], 
+    ],
     "layout": {
-        "title": "Silicon Photovoltaics Learning Curve", 
+        "title": "Silicon Photovoltaics Learning Curve",
         "titlefont": {
-            "color": "", 
-            "family": "", 
+            "color": "",
+            "family": "",
             "size": 0
-        }, 
+        },
         "font": {
-            "family": "Arial, sans-serif", 
-            "size": 12, 
+            "family": "Arial, sans-serif",
+            "size": 12,
             "color": "#000"
-        }, 
-        "showlegend": false, 
-        "autosize": false, 
-        "width": 800, 
-        "height": 440, 
+        },
+        "showlegend": false,
+        "autosize": false,
+        "width": 800,
+        "height": 440,
         "xaxis": {
-            "title": "Cumulative Production (GW)", 
+            "title": "Cumulative Production (GW)",
             "titlefont": {
-                "color": "", 
-                "family": "", 
+                "color": "",
+                "family": "",
                 "size": 0
-            }, 
+            },
             "range": [
-                -3.011967491973726, 
+                -3.011967491973726,
                 2.1561305597186564
-            ], 
+            ],
             "domain": [
-                0, 
+                0,
                 1
-            ], 
-            "type": "log", 
-            "rangemode": "normal", 
-            "showgrid": true, 
-            "zeroline": true, 
-            "showline": false, 
-            "autotick": true, 
-            "nticks": 40, 
-            "ticks": "", 
-            "showticklabels": true, 
-            "tick0": 0, 
-            "dtick": "D1", 
-            "ticklen": 5, 
-            "tickwidth": 1, 
-            "tickcolor": "#000", 
-            "tickangle": 0, 
+            ],
+            "type": "log",
+            "rangemode": "normal",
+            "showgrid": true,
+            "zeroline": true,
+            "showline": false,
+            "autotick": true,
+            "nticks": 40,
+            "ticks": "",
+            "showticklabels": true,
+            "tick0": 0,
+            "dtick": "D1",
+            "ticklen": 5,
+            "tickwidth": 1,
+            "tickcolor": "#000",
+            "tickangle": 0,
             "tickfont": {
-                "family": "", 
-                "size": 0, 
+                "family": "",
+                "size": 0,
                 "color": ""
-            }, 
-            "exponentformat": "e", 
-            "showexponent": "all", 
-            "gridcolor": "#ddd", 
-            "gridwidth": 1, 
-            "zerolinecolor": "#000", 
-            "zerolinewidth": 1, 
-            "linecolor": "#000", 
-            "linewidth": 1, 
-            "anchor": "y", 
-            "position": 0, 
-            "mirror": true, 
-            "overlaying": false, 
+            },
+            "exponentformat": "power",
```

Comment:
ah. there is it :mag:

Reply:
heh yeah, sorry - I should have split out linting those trailing spaces (we don't test that in the mocks?) into its own commit.